src/components/Book.js

```diff
@@ -0,0 +1,42 @@
+// Individual Book component
+// import React from 'react';
```

Comment:
- [ ] It is unprofessional to leave commented code in your codebase, if this highlighted code is not required for the functioning of your project i kindly suggest you remove it :+1: :sparkles: